File: ts/project/.sdk/tm/php/utility/MakeContext.php
<?php
declare(strict_types=1);

// ProjectName SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class ProjectNameMakeContext
{
    public static function call(array $ctxmap, ?ProjectNameContext $basectx): ProjectNameContext
    {
        return new ProjectNameContext($ctxmap, $basectx);
    }
}

File: ts/project/.sdk/tm/csharp/utility/MakeContext.cs
// ProjectName SDK utility: makeContext.

namespace ProjectNameSdk.Util;

public static partial class SdkUtility
{
    internal static Context MakeContextUtil(Dictionary<string, object?>? ctxmap, Context? basectx)
    {
        return new Context(ctxmap, basectx);
    }
}
